posthog/tasks/usage_report.py

```diff
@@ -510,6 +511,43 @@ def get_teams_with_recording_count_in_period(
     return result
 
 
+@timed_log()
+@retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
+def get_teams_with_mobile_billable_recording_count_in_period(begin: datetime, end: datetime) -> list[tuple[int, int]]:
+    previous_begin = begin - (end - begin)
+
+    result = sync_execute(
+        """
+        SELECT team_id, count(distinct session_id) as count
+        FROM (
+            SELECT any(team_id) as team_id, session_id
+            FROM session_replay_events
+            WHERE min_first_timestamp BETWEEN %(begin)s AND %(end)s
+            GROUP BY session_id
+            HAVING (ifNull(argMinMerge(snapshot_source), '') == 'mobile'
+            AND ifNull(argMinMerge(snapshot_library), '') IN (['posthog-ios', 'posthog-android', 'posthog-react-native']))
```

Comment:
shall we right a test with one of each type (web, null, posthog-ios, etc)

i think this `(['posthog-ios', 'posthog-android', 'posthog-react-native'])` syntax isn't right (I don't think you need the square braces)

but let's be sure

Reply:
oh, there is one 🤣

Reply:
```suggestion
            AND ifNull(argMinMerge(snapshot_library), '') IN ('posthog-ios', 'posthog-android', 'posthog-react-native'))
```

Reply:
in which case I think you can remove the square braces anyway...